genie-common-internal/src/main/java/com/netflix/genie/common/internal/dtos/v4/converters/JobDirectoryManifestProtoConverter.java

```diff
@@ -71,6 +71,7 @@ public AgentManifestMessage manifestToProtoMessage(
         return AgentManifestMessage.newBuilder()
             .setJobId(claimedJobId)
             .setManifestJson(manifestJsonString)
+            .setLargeFilesSupported(true)
```

Comment:
Do you view this as a temporary measure or something we'll have to do forever?

Reply:
Transitional. If all clients are running a recent enough version, this can be removed.